suites/pacific/upgrades/tier-1_upgrade_test-4x-to-5x-container_dmcrypt.yaml

```diff
@@ -0,0 +1,152 @@
+#############################################################################
+# - Automation support for containerised upgrade from RHCS4 to RHCS5 cluster
+#
+# Cluster configuration: (conf/pacific/upgrades/upgrades.yaml)
+# --------------------------------------------------------------------------
+# Nodes:
+#   - 3 MONS, 3 MGRs, 3 OSDs, 2 RGW, 2 ISCSIGW's, 1 MDS's, 1 CLIENT, 1 GRAFANA, 1 CLIENT
+#
+# Test steps:
+# ---------------------------------------------------------------------
+# - Deploy containerised Nautilus Ceph cluster using CDN RPMS
+# - Run some I/O's
+# - Upgrade to Pacific using ceph-ansible and parallel run I/O's
+# - adopt to cephadm using ceph-adopt playbook
+# - Run some I/O's
+#
+#############################################################################
+tests:
+- test:
+    name: install ceph pre-requisites
+    module: install_prereq.py
+    abort-on-fail: True
+
+- test:
+    name: ceph ansible install containerized rhcs 4.x from cdn
+    polarion-id: CEPH-83573588
```

Comment:
Since this is a prestep for upgrade and also that relevant polarion ID is already updated in a different test suite where 4x CDN install is being tested as well, this polarion-id could be removed.

Reply:
done i have added a different tc here